website/site/content/download/personal-edition/ubuntu.md

```diff
@@ -147,6 +146,42 @@ Change the dbconfig setting to use the postgres database you created:
 "dbconfig": "postgres://boardsuser:boardsuser-password@localhost/boards?sslmode=disable&connect_timeout=10",
 ```
 
+## Install MySQL 
+
+As an alternative to Postgres, you also can store your data in a MySQL/MariaDB database.
+To install, run:
+
+```
+sudo apt install mariadb-server mariadb-client
+```
+
+Log in as `root` in your database:
+```
+sudo mysql 
+```
+
+At the MySQL prompt, run the following commands (change `user/password` to your own values):
+```
+CREATE DATABASE boards;
+GRANT ALL on boards.* to <b>'boardsuser'@'localhost'</b> identified by '<b>boardsuser-password</b>';
+```
+
+Exit the mysql-prompt:
+```
+exit
+```
+
+Edit the Focalboard `config.json`:
+
+```
+nano /opt/focalboard/config.json
+```
+
+Change the dbconfig setting to use the MySQL database you created:
+"dbtype": "mysql",
+"dbconfig": "boardsuser:boardsuser-password@tcp(127.0.0.1:3306)/boards",
+```,
```

Comment:
And I think this `,` here is not necesary, also the code block is going to be there and the `,` is going to be in the next line, I would remove that.